courses/free-courses-en.md

```diff
@@ -823,6 +823,7 @@
 * [Introduction To Linux](https://www.edx.org/course/introduction-to-linux/) - The Linux Foundation (edx)
 * [Linux Administration Tutorial Videos](https://www.youtube.com/playlist?list=PL9ooVrP1hQOH3SvcgkC4Qv2cyCebvs0Ik) - edureka!
 * [Linux Command Line](https://www.udemy.com/course/command-line/) - Adam Eubankas (Udemy)
+* [Linux Tutorial by Ryan's Tutorials](https://ryanstutorials.net/linuxtutorial)
```

Comment:
Please add the author's name

Reply:
Don't invent titles! Use the title given by the resource. In this case, there's no obviious title, except in the html title element, so...